helper.go

```diff
@@ -724,8 +727,7 @@ func parseRawField(logger Logger, data []byte, msg string) (interface{}, int, er
 func parseUint64(bytes []byte) (ret uint64, err error) {
```

Comment:
This still has named return values

Reply:
There were a few remaining indeed that need to be addressed. Is it acceptable to iterate on this in another PR?

Reply:
Sure, sounds fine.